lib/ExpensiMark.js

```diff
@@ -163,11 +163,19 @@ export default class ExpensiMark {
                 replacement: '<br />',
             },
             {
-                // when <pre> and <br /> occur together extra line is added so removing <br />
+                //eslint-disable-next-line no-useless-escape
+                // We're removing <br /> because when </pre> and <br /> occur together, an extra line is added.
                 name: 'replacepre',
-                regex: /<\/pre>\s*<br\s*[/]?>/gi,
+                regex: /<\/pre>\s*<br\s*[\/]?>/gi,
                 replacement: '</pre>',
             },
+            {
+                //eslint-disable-next-line no-useless-escape
+                // We're removing <br /> because when <br /> and <pre> occur together, an extra line is added.
+                name: 'replacebr',
+                regex: /<br\s*[\/]?><pre>\s*/gi,
```

Comment:
The escape character is not needed here (https://regexr.com/6k3g3)

```suggestion
                regex: /<br\s*[/]?><pre>\s*/gi,
```

and the eslint error `no-useless-escape` won't be thrown

Reply:
@marcochavezf removed the escape character. Let me know if this is alright and apologies for the back and forth of this PR. Thanks

Reply:
Thanks @aneequeahmad! No problem, once this PR is merged we'll need another a separate PR in E/App to update the `package.json`

Reply:
Great, looking forward to it.